frontend/src/App.tsx

```diff
@@ -23,10 +24,26 @@ const App = (): ReactElement => {
         <Route exact path="/login" component={Login} />
         <Route path="/sign_up" component={CreateUser} />
         <AuthenticatedRoute path="/dashboard" component={Dashboard} />
-        <AuthenticatedRoute exact path="/interventions" component={Interventions} />
-        <AuthenticatedRoute exact path="/interventions/:id" component={Activities} />
-        <AuthenticatedRoute exact path="/assessments/:id" component={Quiz} />
-        <AuthenticatedRoute path="/assessments" component={Assessments} />
+        <AuthenticatedRoute
+          exact
+          path="/interventions"
+          component={Interventions}
+        />
+        <AuthenticatedRoute
+          exact
+          path="/interventions/:id"
+          component={Activities}
+        />
+        <AuthenticatedRoute
+          exact
+          path="/assessments/:id"
+          component={Quiz}
+        />
+        <AuthenticatedRoute path="/reward" component={Reward} />
```

Comment:
I think this path should be "/interventions/reward", to be clear what the reward is for.

Reply:
Ah should I do this part with this week ticket -> hooks this up to assessment

Reply:
Idk it is a super quick change that could be done in this pr tbh.

Reply:
yessir ill change it xD